src/Components/Publishing/EditorialFeature/Components/Vanguard2019/Components/TOC.tsx

```diff
@@ -0,0 +1,12 @@
+import { Box, Sans } from "@artsy/palette"
+import React from "react"
+
+export const VanguardTOC: React.SFC<{
```

Comment:
I wonder if we should rename this to use Table of Contents. Looking at the file name it wasn't immediately clear to me what TOC was.